birdhouse/components/scheduler/config.yml.template

```diff
@@ -1,54 +1,9 @@
 ---
-- name: logrotate
-  comment: Rotate log files under ${BIRDHOUSE_LOG_DIR}
-  schedule: '@daily'
-  command: bash -c 'cp /etc/logrotate.conf.orig /etc/logrotate.conf && chown root:root /etc/logrotate.conf && chmod 644 /etc/logrotate.conf && /usr/sbin/logrotate -v /etc/logrotate.conf'
-  dockerargs: >-
-    --rm --name logrotate
-    --volume ${BIRDHOUSE_LOG_DIR}:/var/log/birdhouse:rw
-    --volume ${BIRDHOUSE_LOGROTATE_DATA_DIR}:/var/lib:rw
-    --volume ${COMPOSE_DIR}/deployment/birdhouse-deploy.logrotate:/etc/logrotate.conf.orig:ro
-  image: 'stakater/logrotate:3.13.0'
-
-- name: notebookdeploy
-  comment: Auto-deploy tutorial notebooks
-  schedule: '${BIRDHOUSE_AUTODEPLOY_NOTEBOOK_FREQUENCY}'
-  command: '${COMPOSE_DIR}/deployment/trigger-deploy-notebook'
-  dockerargs: >-
-    --rm --name notebookdeploy
-    --volume /var/run/docker.sock:/var/run/docker.sock:ro
-    --volume ${BIRDHOUSE_LOG_DIR}:${BIRDHOUSE_LOG_DIR}:rw
-    --volume ${COMPOSE_DIR}:${COMPOSE_DIR}:ro
-    --volume ${JUPYTERHUB_USER_DATA_DIR}:${JUPYTERHUB_USER_DATA_DIR}:rw
-    --volume /tmp/notebookdeploy:/tmp/notebookdeploy:rw
-    --volume ${BIRDHOUSE_LOCAL_ENV}:${BIRDHOUSE_LOCAL_ENV}
-    --env COMPOSE_DIR=${COMPOSE_DIR}
-    --env TMP_BASE_DIR=/tmp/notebookdeploy
-    --env BIRDHOUSE_LOCAL_ENV=${BIRDHOUSE_LOCAL_ENV}
-    --env BIRDHOUSE_COMPOSE=${BIRDHOUSE_COMPOSE}
-    --env JUPYTERHUB_USER_DATA_DIR=${JUPYTERHUB_USER_DATA_DIR}${AUTODEPLOY_NOTEBOOK_EXTRA_DOCKER_ARGS}
-  image: 'docker:19.03.6-git'
-
-- name: autodeploy
-  comment: Auto-deploy entire Birdhouse platform
-  schedule: '${BIRDHOUSE_AUTODEPLOY_PLATFORM_FREQUENCY}'
-  command: '${COMPOSE_DIR}/deployment/triggerdeploy.sh ${COMPOSE_DIR}'
-  dockerargs: >-
-    --rm --name autodeploy${BIRDHOUSE_AUTODEPLOY_EXTRA_REPOS_AS_DOCKER_VOLUMES}
-    --volume /var/run/docker.sock:/var/run/docker.sock:ro
-    --volume ${BIRDHOUSE_LOG_DIR}:${BIRDHOUSE_LOG_DIR}:rw
-    --volume ${COMPOSE_DIR}/..:${COMPOSE_DIR}/..:rw
-    --volume ${BIRDHOUSE_AUTODEPLOY_DEPLOY_KEY_ROOT_DIR}:${BIRDHOUSE_AUTODEPLOY_DEPLOY_KEY_ROOT_DIR}:ro
-    --volume ${JUPYTERHUB_USER_DATA_DIR}:${JUPYTERHUB_USER_DATA_DIR}:rw
-    --volume ${BIRDHOUSE_LOCAL_ENV}:${BIRDHOUSE_LOCAL_ENV}
```

Comment:
This is the previous code, there are no `$` escape.  Is there any reasons to start escaping now?

Reply:
Yes.. see https://github.com/bird-house/birdhouse-deploy/pull/503#discussion_r1991448930

Reply:
This is no longer relevant since [b923730](https://github.com/bird-house/birdhouse-deploy/pull/503/commits/b923730b6b8c1befd35677bab7946444c794e63a)